src/App.js

```diff
@@ -45,8 +20,8 @@ function App() {
           <NavBar name={User.name} initials={User.initials} color={User.color}/>
```

Comment:
Instead of importing `User` here and passing it as prop to navbar, you could just import `User` from navbar directly.
Same for your `Profile` component.

A great alternative is using a context to store your user state so it is globally accessible

Reply:
ok you actually have created a `UserContext` - highly recommend using that